src/vs/workbench/parts/debug/common/debugModel.ts

```diff
@@ -490,6 +490,18 @@ export class Thread implements IThread {
 		});
 	}
 
+	/**
+	 * Returns exception info promise if the exception was thrown and the debug adapter supports 'exceptionInfo' request, otherwise null
+	 */
+	public get exceptionInfo(): TPromise<DebugProtocol.ExceptionInfoResponse> {
+		const session = this.process.session;
+		if (session.capabilities.supportsExceptionInfoRequest && this.stoppedDetails && this.stoppedDetails.reason === 'exception') {
+			return session.exceptionInfo({ threadId: this.threadId });
+		}
+
+		return null;
```

Comment:
You can not just return `null` when a promise is expected, you should
`return TPromise.as(null)`